Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/ColorManagement/TransformColor.azsli

```diff
@@ -13,16 +13,22 @@
 #include "GeneratedTransforms/LinearSrgb_To_Srgb.azsli"
 #include "GeneratedTransforms/Srgb_To_LinearSrgb.azsli"
 #include "GeneratedTransforms/Aces_To_AcesCg.azsli"
+#include "GeneratedTransforms/LinearColor_To_Hsv.azsli"
+#include "GeneratedTransforms/Hsv_To_LinearColor.azsli"
+#include "GeneratedTransforms/AcesCc_To_AcesCg.azsli"
+#include "GeneratedTransforms/AcesCg_To_AcesCc.azsli"
 #include "GeneratedTransforms/CalculateLuminance_LinearSrgb.azsli"
 #include "GeneratedTransforms/CalculateLuminance_AcesCg.azsli"
 
 enum class ColorSpaceId
 {
     SRGB = 0,
     LinearSRGB,
+    ACEScc,
     ACEScg,
     ACES2065,
     XYZ,
+    HSV,
```

Comment:
It should be correct to apply the same equation to any linear color space right?

Reply:
The other enums represent specific color spaces though. If you're in "HSV" how would you know if it's an sRGB version of HSV or ACEScg, or XYZ, etc? They all have different primaries. HSV is just a different way of looking at an rgb style color space. There are other transforms that are similar, for instance YCoCg is another way of looking at an rgb color space.

Thinking about this more I think I'd recommend not even going through TransfromColor() for an HSV transform. It's just not used the same way as full color space transforms. It'd be better to just call LinearColorToHsv() and HsvToLinearColor() because it's up to the caller to know the actual space you're working in.

Reply:
Yes I agree, HSV is a color model, not a color space.